integrations/LaunchDarkly/browser.js

```diff
@@ -0,0 +1,84 @@
+import {
+    isArray,
+    isObject,
+} from "../../utils/utils"; import logger from "../../utils/logUtil";
+import ScriptLoader from "../ScriptLoader";
+const isString = function isString(val) {
+    return toStr$1.call(val) === "[object String]";
+};
+class LaunchDarkly {
+    constructor(config) {
+        this.name = "LaunchDarkly";
+        this.clientSideId = config.clientSideId;
+    }
+    init() {
+        logger.debug("===in init LaunchDarkly===");
+        if (!this.clientSideId) {
+            logger.debug("clientSideId is missing");
```

Comment:
This should be an error.
```suggestion
            logger.error(`${this.name} :: Unable to initialize destination - clientSideId is missing in config`);
```

Reply:
Changed to error.

Reply:
Change the error message to make it clear.